app/uk/gov/hmrc/cdsreimbursementclaimfrontend/controllers/claims/EnterClaimController.scala

```diff
@@ -67,9 +66,9 @@ class EnterClaimController @Inject() (
 
   implicit val dataExtractor: DraftC285Claim => Option[ClaimsAnswer] = _.claimsAnswer
 
-  def startClaim(): Action[AnyContent] =
+  def startClaim(implicit journeyBindable: JourneyBindable): Action[AnyContent] =
```

Comment:
why is this parameter an implicit? where does it implicitly get picked up from?

Reply:
`withAnswersAndRoutes[ClaimsAnswer]`